File: semester I/Praktikum Pemrograman I/Modul6_JovanGilbertNatamasindah_2310817310002/Python/PRAK602-2310817310002-JovanGilbertNatamasindah.py
import numpy as np

TOTAL = int(input("Masukkan Total bilangan yang akan diinputkan : "))

ELEMEN_ARRAY = list(map(int,input("Masukkan angka matriks : ").split()))

arr = np.array(ELEMEN_ARRAY).reshape((1,TOTAL))

i = 0

print("Hasil : ")

while i < TOTAL :
    HASIL = ELEMEN_ARRAY[i] * (i + 1)
    print(f"{HASIL}", end = " ")
    i += 1

File: semester I/Praktikum Pemrograman I/Modul6_JovanGilbertNatamasindah_2310817310002/C/PRAK602-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>

void main() {
    int TOTAL;

    printf("Masukkan Total bilangan yang akan diinputkan : ");
    scanf("%d", &TOTAL);
    
    int arr[TOTAL];

    printf("Masukkan angka : ");
    for(int i = 0 ; i < TOTAL ; i++) {
        scanf("%d", &arr[i]);
    }

    printf("Hasil : ");

    for(int i = 0 ; i < TOTAL ; i++) {
        arr[i] = arr[i] * (1 + i);
        printf("%d ", arr[i]);
    }
}

